presentation/src/main/java/com/safetyheads/akademiaandroida/presentation/ui/fragments/faq/FaqViewModel.kt

```diff
@@ -0,0 +1,69 @@
+package com.safetyheads.akademiaandroida.presentation.ui.fragments.faq
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.safetyheads.akademiaandroida.domain.entities.faqs.FaqTab
+import com.safetyheads.akademiaandroida.domain.entities.faqs.toFaqType
+import com.safetyheads.akademiaandroida.domain.entities.firebasefirestore.faq.Faq
+import com.safetyheads.akademiaandroida.domain.entities.firebasefirestore.faq.Question
+import com.safetyheads.akademiaandroida.domain.entities.firebasefirestore.faq.Type
+import com.safetyheads.akademiaandroida.domain.usecases.AddQuestionUseCase
+import com.safetyheads.akademiaandroida.domain.usecases.GetFaqUseCase
+import kotlinx.coroutines.launch
+
+class FaqViewModel(
+    private val getFaqUseCase: GetFaqUseCase,
+    private val addQuestionUseCase: AddQuestionUseCase
+) : ViewModel() {
+
+    private var selectedType: Type
+
+    private val _typedFaqsList: MutableLiveData<List<Faq>?> = MutableLiveData<List<Faq>?>().apply {
```

Comment:
List<Faq>? does it need to be nullable? We can always pass an empty list if there is no data